### PR TITLE
Update connect-to-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Made cahnges as installing the module does not need elevated rights.

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9491

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ For procedures that require the new cmdlets in the Azure Active Directory PowerS
 
 These steps are required only one time on your computer. But you'll likely need to update the software periodically.
   
-1. Open an elevated Windows PowerShell Command Prompt window (run Windows PowerShell as an administrator).
+1. Open Windows PowerShell Command Prompt window.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
1. Open a Windows PowerShell Command Prompt window.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**